core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java

```diff
@@ -0,0 +1,787 @@
+package com.klaytn.caver.tx.kct;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.crypto.KlayCredentials;
+import com.klaytn.caver.methods.response.KlayLogs;
+import com.klaytn.caver.methods.response.KlayTransactionReceipt;
+import com.klaytn.caver.tx.SmartContract;
+import com.klaytn.caver.tx.manager.TransactionManager;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import org.web3j.abi.FunctionEncoder;
+import org.web3j.abi.TypeReference;
+import org.web3j.abi.datatypes.Address;
+import org.web3j.abi.datatypes.Bool;
+import org.web3j.abi.datatypes.Event;
+import org.web3j.abi.datatypes.Function;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.abi.datatypes.Utf8String;
+import org.web3j.abi.datatypes.generated.Uint256;
+import org.web3j.abi.datatypes.generated.Uint8;
+import org.web3j.protocol.core.RemoteCall;
+import org.web3j.tx.gas.ContractGasProvider;
+
+
+public class KIP7 extends SmartContract {
+    /*
+    * KIP-7 token contract source code
+    * caver-js/packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7Token.sol
+    *
+    * The ABI and bytecode below are built via the following command.
+    *  solc --abi --bin --allow-paths . ./packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7Token.sol
```

Comment:
I think this path(./packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7Token.sol) should be edited.